manage.py

```diff
@@ -4,6 +4,8 @@
 
 
 if __name__ == '__main__':
+    if (sys.version_info[0], sys.version_info[1]) < (3, 6):
+        raise Exception('Minimum python version v3.6 is required.')
```

Comment:
`version v3.6` is a bit redundant, since the `v` in `v3.6` already means version.

Reply:
I have changed the text to 'Minimum python version 3.6 is required.'